src/index.html

```diff
@@ -9,10 +9,63 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.scss"
+    />
```

Comment:
The stylesheet link is pointing to a SCSS file (`main.scss`). Browsers cannot directly interpret SCSS files. You need to compile this SCSS file into a CSS file and then link the resulting CSS file in your HTML.